engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -112,14 +113,18 @@ def stop_on_any_exception(func):
     def try_catch_handling(self, input_data, parameters):
         try:
             return func(self, input_data, parameters)
-        except Exception as e:
+        except Exception:
             logging.exception("Rolling batch inference error")
-            err = json.dumps({"code": 500, "error": str(e)})
+            err = {
```

Comment:
I don't think this is necessary. When process crashed, these data is not send back anyway

Reply:
this is generic for all rolling batch, if python exception reached, we send back all

Reply:
I don't think we should send stacktrace to end user.